From e3e2f2dc8f693119ff40e5366d5bd9bc590eeffb Mon Sep 17 00:00:00 2001
From: Stefano Rivera <stefanor@debian.org>
Date: Wed, 13 Mar 2019 14:50:37 -0700
Subject: [PATCH] Put pypy namespace files in the correct place:
 /usr/lib/pypy/ns.

---
 debian/changelog | 6 ++++++
 dh_pypy          | 2 +-
 2 files changed, 7 insertions(+), 1 deletion(-)

diff --git a/debian/changelog b/debian/changelog
index d29b1e4..2ac9ba6 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,9 @@
+dh-python (3.20190313) UNRELEASED; urgency=medium
+
+  * Put pypy namespace files in the correct place: /usr/lib/pypy/ns.
+
+ -- Stefano Rivera <stefanor@debian.org>  Wed, 13 Mar 2019 14:47:03 -0700
+
 dh-python (3.20190308) unstable; urgency=medium
 
   * so2pyver: add a fallback to UTF-8 if locale.getdefaultlocale() returns None
diff --git a/dh_pypy b/dh_pypy
index b16487f..97bee8b 100755
--- a/dh_pypy
+++ b/dh_pypy
@@ -282,7 +282,7 @@ def main():
                 log.error('cannot remove __init__.py from package: %s', e)
                 exit(6)
         if nsp:
-            dstdir = join('debian', package, 'usr/share/pypy/ns/')
+            dstdir = join('debian', package, 'usr/lib/pypy/ns/')
             if not exists(dstdir):
                 os.makedirs(dstdir)
             with open(join(dstdir, package), 'a', encoding='utf-8') as fp:
-- 
2.20.1

